packages/react/src/components/ValueCard/_value-renderer.scss

```diff
@@ -18,6 +18,12 @@
     line-height: $line-height;
     @include multiline-text-overflow($max-lines, $line-height);
     word-break: break-all;
+    /* override default button styles to match span visuals when using a button */
+    &.#{$prefix}--btn {
+      padding: 0;
```

Comment:
I tried. The problem is even the smallest padding (`$spacing-01`) causes truncation and changes what's visible compared to next. I could add a title (which isn't there currently when item are truncated), but not sure if the padding is worth it, TBH. I'm certainly open to input though...

<img width="1253" alt="Screen Shot 2021-09-02 at 7 44 03 AM" src="https://user-images.githubusercontent.com/609466/131838022-51a3dbc6-72a7-4f81-a9b6-c7fa5ffb42d5.png">

Reply:
I see. Perhaps we should ask a designer, possibly skip the focused outline all together here and indicate it by background color or something?

Reply:
@JordanWSmith15 Do you have any input here, or want to reach out to Ben to ask?

Reply:
@davidicus I'm not sure I follow. I'm using a `<Button kind="ghost"/>` element when they pass the onAttributeClick callback, but the problem is the default padding on the button throws everything off, so I removed that. That still meet a11y, right? 

We're just trying to figure out the visuals of what to do with no padding/outlines styles on focus.

Reply:
Removing the padding meets a11y but having only a color change denote a change in state does not so the outline is necessary. My suggestion is around using the `outline` property to produce that outline instead of the `border` property. 

With `outline` property we can provide an offset which you can perhaps use to provide a padding between the button and the outline so it looks more visually appealing.